Tests/MongoSwiftTests/MongoDatabaseTests.swift

```diff
@@ -9,13 +9,17 @@ final class MongoDatabaseTests: MongoSwiftTestCase {
         ]
     }
 
+    override class func testDatabase() -> String {
+        return "testDB"
+    }
+
     override func setUp() {
         continueAfterFailure = false
     }
 
     func testDatabase() throws {
```

Comment:
I just noticed that this has the same name as the added function (or computed property if my early suggestion actually works).

Let's rename this to something like `testMongoDatabase()`?

Reply:
Good catch, updated to `testMongoDatabase`.